pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -502,10 +519,14 @@ func (p *provider) Cleanup(machine *v1alpha1.Machine, _ *cloud.MachineCreateDele
 		return false, osErrorToTerminalError(err, "failed to get compute client")
 	}
 
-	if err := osservers.Delete(computeClient, instance.ID()).ExtractErr(); err != nil {
+	if err := osservers.Delete(computeClient, instance.ID()).ExtractErr(); err != nil && err.Error() != "Resource not found" {
```

Comment:
`isNotFoundErr(err)` is in this very package.

Reply:
You have a link?

Reply:
@kdomanski I can not find anything by that name in either this repo or gophercloud/gophercloud